dirty_cat/super_vectorizer.py

```diff
@@ -161,13 +198,16 @@ def __init__(self, *,
         self.datetime_transformer = datetime_transformer
         self.auto_cast = auto_cast
         self.impute_missing = impute_missing
+        self.handle_unknown = handle_unknown
 
         self.remainder = remainder
         self.sparse_threshold = sparse_threshold
         self.n_jobs = n_jobs
         self.transformer_weights = transformer_weights
         self.verbose = verbose
 
+        self.low_card_cat_transformer = _set_handle_unknown(self.low_card_cat_transformer, handle_unknown)
```

Comment:
By scikit-learn conventions, no input values should be modified during "__init__". This line should be moved to the beginning of fit (that way, if "handle_unknown" is modified after the init, it gets propagated).